array-loops.js

```diff
@@ -3,30 +3,43 @@
 // --------------------------
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
```

Comment:
Hoi Loredana, dit kun je veel algemener maken, stel je hebt 1000 waardes in dit array staan...